frontend/src/components/widgets/DataFrame/hooks/useColumnLoader.ts

```diff
@@ -38,17 +38,29 @@ export const INDEX_IDENTIFIER = "index"
 // Prefix used in the config column mapping when referring to a column via the numeric position
 export const COLUMN_POSITION_PREFIX = "col:"
 
+export const COLUMN_WIDTH_MAPPING = {
+  small: 75,
```

Comment:
Should these be capitalized since they're constants? I would think that maybe these are lowercase because they're supposed to be similar to theme but it feels like they should be uppercase.

Reply:
They are lowercased to match the user provide config. I will add a comment for this.